File: PHP/1052.php
<?php
// Problema: 1052 - Mês         | Resposta: Accepted
// Linguagem: PHP (8.1.2) [+1s] | Tempo: 0.045s

$mes = intval(trim(fgets(STDIN)));
$dt = DateTime::createFromFormat('!m', $mes);
echo "{$dt->format('F')}\n";
?>

File: C++/1052.cpp
// Problema: 1052 - Mês   | Resposta: Accepted
// Linguagem: C20++ [+0s] | Tempo: 0.000s

#include <iostream>
#include <array>
#include <string>

int main() {
    std::array<std::string, 12> meses = {
        "January", "February", "March",
        "April", "May", "June",
        "July", "August", "September",
        "October", "November", "December"
    };

    int mes;
    std::cin >> mes;

    std::cout << meses[mes - 1] << '\n';

    return 0;
}